test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -0,0 +1,164 @@
+
+        kieServerControllerClientProvider = new KieServerControllerClientProvider(workbenchWithKieServerScenario.getWorkbenchDeployment());
+        kieServerMgmtControllerClient = kieServerControllerClientProvider.getKieServerMgmtControllerClient();
+        kieServerControllerClientProvider.waitForServerTemplateCreation();
+    }
+
+    @After
+    public void tearDown() {
+        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getWorkbenchDeployment());
+        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getKieServerDeployment());
+        workbenchWithKieServerScenario.undeploy();
+        gitProvider.deleteGitRepository(workbenchWithKieServerScenario.getNamespace());
+    }
+
+    @Test
+    public void kieServerScaleTest() {
+        KieServerInfo serverInfo = kieServerClientProvider.getKieServerClient().getServerInfo().getResult();
+        checkServerTemplateIsRegistred(serverInfo.getServerId(), serverInfo.getName());
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        scaleKieServerTo(1, serverInfo.getServerId());
+
+        logger.debug("Register Kie Container to Kie Server");
+        kieServerMgmtControllerClient.saveContainerSpec(serverInfo.getServerId(), serverInfo.getName(), CONTAINER_ID, CONTAINER_ALIAS, PROJECT_GROUP_ID, PROJECT_NAME, PROJECT_VERSION, KieContainerStatus.STARTED);
+        kieServerClientProvider.waitForContainerStart(CONTAINER_ID);
+
+        logger.debug("Start process instance");
+        Long signalPid = kieServerClientProvider.getProcessClient().startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+        Assertions.assertThat(signalPid).isNotNull().isGreaterThan(0L);
+        Assertions.assertThat(kieServerClientProvider.getProcessClient().findProcessInstances(CONTAINER_ID, 0, 10)).isNotNull().hasSize(1);
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        logger.debug("Start process instance");
+        try {
+            Long noContainerPid = kieServerClientProvider.getProcessClient().startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+            Assertions.fail("Container was registred on not existing server template with id " + noContainerPid);
+        } catch (KieServerHttpRequestException ex) {
+            // ok - requested URL does not exists
+            logger.debug("KieServerHttpRequestException catched:\n" + ex.toString());
+        } catch (KieServicesHttpException ex) {
+            // ok - response code 403 Forbidden
+            // The application is currently not serving requests at this endpoint. It may not have been started or is still starting.
+            logger.debug("KieServicesHttpException catched:\n" + ex.toString());
+        }
+
+        scaleKieServerTo(1, serverInfo.getServerId());
+
+        logger.debug("Wait for Kie Container start again");
+        kieServerClientProvider.waitForContainerStart(CONTAINER_ID);
```

Comment:
Not needed. Managed Kie server is ready when it installs all containers defined in server template. This is checked in previous step - scaleKieServerTo.